suite-native/navigation/src/components/ScreenSubHeader.tsx

```diff
@@ -8,30 +8,29 @@ import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { GoBackIcon } from './GoBackIcon';
 import { CloseActionType } from '../navigators';
 
-type ScreenSubHeaderProps = RequireOneOrNone<
+export type ScreenSubHeaderProps = RequireOneOrNone<
     {
         content?: ReactNode;
         rightIcon?: ReactNode;
         leftIcon?: ReactNode;
         closeActionType?: CloseActionType;
+        customHorizontalPadding?: number;
     },
     'leftIcon' | 'closeActionType'
 >;
 
 const ICON_SIZE = 48;
 
-const headerStyle = prepareNativeStyle(utils => {
-    const padding = utils.spacings.small;
-
-    return {
+const headerStyle = prepareNativeStyle<{ customHorizontalPadding?: number }>(
+    (utils, { customHorizontalPadding }) => ({
         flexDirection: 'row',
         justifyContent: 'space-between',
         alignItems: 'center',
-        padding,
+        padding: customHorizontalPadding || utils.spacings.small,
         backgroundColor: utils.colors.backgroundSurfaceElevation0,
-        height: ICON_SIZE + padding * 2,
-    };
-});
+        height: ICON_SIZE + utils.spacings.medium,
```

Comment:
Imo it was more understandable why to have this extra value with `padding * 2`

Reply:
fixed (already squashed)